industrial_ci/ci_main.sh

```diff
@@ -325,4 +325,4 @@ cd $TARGET_REPO_PATH  # cd back to the repository's home directory with travis
 pwd
 
 HIT_ENDOFSCRIPT=true
-success 0
```

Comment:
I had to think twice why this would terminate the travis job. Then I noticed that we source all scripts..